NetscapeBookmarkParser.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/**
+ * Generic Netscape bookmark parser
+ */
+class NetscapeBookmarkParser
+{
+    protected $defaultTag = '';
+    protected $defaultPub = false;
+    protected $items = array();
```

Comment:
Default values are useless for attributes set in the constructor, which is a best practice.